.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
can we please use ESM style module syntax if possible?

Reply:
This link : https://eslint.org/docs/user-guide/configuring/configuration-files 
says that :ESLint does not support ESM configuration at this time

Reply:
ok makes sense.